src/components/header/Header.tsx

```diff
@@ -0,0 +1,26 @@
+import { observer } from "mobx-react-lite"
+import "./Header.scss"
+import { Button } from "../button/Button"
+
+interface IHeaderProps {
+    isSideMenuVisible?: () => void
+}
+
+export const Header: React.FC<IHeaderProps> = observer(({ isSideMenuVisible }) => {
+    return (
+        <div className="header">
+            <Button
+                leftIcon={
```

Comment:
Za ikone imava icons folder. 

Ikono daj v icons folder in mora biti wrappana v IconBase. Poglej si primer za DashboardIcon.

Uporabi pol ikono kot komponento iz icons.

Reply:
Se pravi, da po tem takem niti nebi rabil wrappat ikono v Button? Bi lahko dodal samo ikono ki je SideMenuButton komponenta?

Reply:
Ja pa ne. Je pa res da Button ni menda pravo mesto za to naredit. Najboljše je da narediš novo komponento `ButtonIcon` ki sprejme children, ki bo ikona, pa ji da ta dodaten button icon styling zraven.